test_requirements.txt

```diff
@@ -49,6 +49,8 @@ redis~=4.1
 requests-mock~=1.6
 requests~=2.20
 s3fs>=0.3.0, <0.5  # Needs to be at least 0.3.0 to make use of `cachable` attribute on S3FileSystem.
+scikit-learn~=1.1
+scipy~=1.9
```

Comment:
You'll also need to add this dataset to [`setup.py`](https://github.com/kedro-org/kedro/blob/main/setup.py) together with those requirements.